mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -25,21 +25,24 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(SparseTensor, sparse_tensor);
 /// These correspond to SparseTensorEncodingAttr::LevelType in the C++ API.
 /// If updating, keep them in sync and update the static_assert in the impl
 /// file.
+
+typedef uint64_t level_type;
+
 enum MlirSparseTensorLevelType {
```

Comment:
so, now the difference is that these still fit in 32-bit?

Reply:
Yes, I kept the enum for building py_enum in the python binding, which will be helpful for us to buildLevelType with enum. WDYT?